libcodechecker/libhandlers/analyze.py

```diff
@@ -89,6 +90,12 @@ def add_arguments_to_parser(parser):
     Add the subcommand's arguments to the given argparse.ArgumentParser.
     """
 
+    def is_ctu_capable():
```

Comment:
We have a detection function implemented for checking if clang exists or not. I don't recall the exact location, but you could find it by starting from `libcodechecker.libhandlers.analyzers` (or `checkers`, even).

This function should be introduced in a similar fashion, under `libcodechecker.analyze` package.